### DISCUSSION TITLE
Question about V10

### DISCUSSION THREAD

**Discussion by pwideman** [timgit/pg-boss#445] [General]
Just a quick question: it looks like the job OnComplete functionality has been removed in v10. Is that going to be permanent? Is the alternative suggestion just to create successive jobs in the job handler functions? I'm currently using v9 and making use of the OnComplete functionality to schedule successive jobs based on the result of the current job. If that is going away in v10 then I'll go ahead and switch to the new pattern now.

Thanks!

**Comment by timgit** [comment]
Yes, the goal in v10 is to standardize on dead letter queues for this behavior.
